AEPAnalytics.xcodeproj/project.pbxproj

```diff
@@ -953,6 +953,7 @@
 		};
 		AC277D05252C328B00987C67 /* ShellScript */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
is functionaltests this the only target that require this setting ?

Reply:
It's just the SDK target that has the run script phase

Reply:
and we are not running it for source builds?

Reply:
We are not running in any test or testApp targets but only in the source build (AEPAnalytics) target.